ipa-metrics/Cargo.toml

```diff
@@ -13,6 +13,11 @@ partitions = []
 crossbeam-channel = "0.5"
 # This crate uses raw entry API that is unstable in stdlib
 hashbrown = "0.15"
+# Open telemetry crates: opentelemetry-prometheus crate implementation is based on Opentelemetry API and SDK 0.23. (TBC)
+opentelemetry = "0.24"
```

Comment:
this is great, but can you make a separate crate for it? `ipa-metrics` is intended to contain only core functionality and it is easier to maintain in the long term if it does not have extra dependencies